tests/integration/s2n_handshake_test_s_client.py

```diff
@@ -120,41 +122,64 @@ def try_handshake(endpoint, port, cipher, ssl_version, sig_algs=None, curves=Non
     return 0
 
 def print_result(result_prefix, return_code):
-    print(result_prefix, end='')
+    suffix = ""
     if return_code == 0:
         if sys.stdout.isatty():
-            print("\033[32;1mPASSED\033[0m")
+            suffix = "\033[32;1mPASSED\033[0m"
         else:
-            print("PASSED")
+            suffix = "PASSED"
     else:
         if sys.stdout.isatty():
-            print("\033[31;1mFAILED\033[0m")
+            suffix = "\033[31;1mFAILED\033[0m"
         else:
-            print("FAILED")
-
+            suffix ="FAILED"
+            
+    print(result_prefix + suffix)
+
+def createThreadPool():
+    threadpoolSize = multiprocessing.cpu_count() * 4
```

Comment:
I set it higher than the number of cores to ensure that it maximizes CPU utilization. On my desktop it looks like performance maxes out at `multiprocessing.cpu_count() * 2`